positivenegative.c

```diff
@@ -0,0 +1,14 @@
+//Write a program to check if a user-input number is positive or negative. Assume the user enters a valid number, but that number may not be an integer. File name is your choice, as long as it makes sense.
```

Comment:
Great job! the only suggestion i have is to include text that will prompt the user.